frontroomsbot/bot.py

```diff
@@ -54,6 +55,26 @@ async def sync(interaction: discord.Interaction):
     await interaction.response.send_message("Synced!")
     print("Command tree synced")
 
+@tree.command(name="csgoroadtoglobal", description="CSGOROADTOGLOBAL", guild=guild)
+async def csgo(interaction: discord.Interaction):
+    # list of zmrds
+    zmrdi = ['JosefKuchar', 'Roytak', 'Soromis', 'TominoBLM', 'Addy Daddy', 'Kubik', 'MrStinky']
```

Comment:
I wish to be the person I was before seeing this

Reply:
I fixed!! It's ok now I think, but please check, master.